### PR TITLE
Add Plone 5 compatibility

### PR DESCRIPTION
**Pull request by laulaz**

### CODE REVIEW

**Comment by laulaz** [comment]
@jenkins-plone-org please run jobs

File: CHANGES.rst [hunk 1]
```diff
@@ -6,7 +6,8 @@ Changelog
 
 Breaking changes:
 
-- *add item here*
+- Add Plone 5 compatibility
+  [laulaz]
```
Comment (jensens) [inline CHANGES.rst]: is this breaking? IMO its a feature. Or does  it break 4.x?

Comment (laulaz) [reply CHANGES.rst]: @jensens OK ... changed 👍 
Indeed, it is not breaking Plone 4 ...

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**